### PR TITLE
Add new prefixes to the "sensitiveBackendPrefixes" list

### PR DESCRIPTION
**Pull request by kimlisa**
#### Description 
When [deleting reset password prefix legacy code](https://github.com/gravitational/teleport/pull/19285), I did a global search of `resetpassword` keyword and found that I did not add `usertoken` prefix (added in `v8`) to the `sensitiveBackendPrefixes` list which is used for diagnostics metrics, which reintroduced this security fix: https://github.com/gravitational/teleport/pull/5793/commits/00a4de5a52843020f63d8a55d82bd2cb09ea4c45

Also adds new webauthn related prefix per @codingllama 

Confirmed that this minor fix doesn't have to be private.

This PR needs to be `rebased and merged` instead of `squash` to retain the commits because:
  - `usertoken` needs backporting back to `v8`
  - `sessionData` needs backporting back to `v9`

### CODE REVIEW

File: lib/backend/report_test.go [hunk 1]
```diff
@@ -84,3 +84,21 @@ func TestBuildKeyLabel(t *testing.T) {
...
+func TestBuildLabelKey_SensitiveBackendPrefixes(t *testing.T) {
+	testCases := []struct {
+		input  string
+		masked string
+	}{
+		{"/tokens/1234-5678", "/tokens/******678"},
+		{"/usertoken/1234-5678", "/usertoken/******678"},
+		{"/access_requests/1234-5678", "/access_requests/******678"},
+
+		{"/webauthn/sessionData/scope/1234-5678", "/webauthn/sessionData"},
```
Comment (kimlisa) [inline lib/backend/report_test.go:97]: i built a key as was used in prod and the `buildKeyLabel` caps the key parts to two parts as stated here in [comments](https://github.com/gravitational/teleport/commit/00a4de5a52843020f63d8a55d82bd2cb09ea4c45#diff-a640ddd8ed0d579e4e5a19f8812542435b20e9024a52599da8b7aeb93fbf8dc1R288)

File: lib/backend/report.go [hunk 2]
```diff
@@ -408,8 +408,8 @@ func buildKeyLabel(key string, sensitivePrefixes []string) string {
 // sensitive values.
 var sensitiveBackendPrefixes = []string{
 	"tokens",
-	"resetpasswordtokens",
-	"adduseru2fchallenges",
+	"usertoken",
+	"sessionData",
```
Comment (kimlisa) [inline lib/backend/report.go:412]: i did attempt to make this a tad more resilient by trying to import the keys, but it created a import cycle problem (`lib/local` imports from `lib/backend` and `lib/backend` imports from `lib/local`), so I left it as is.

Comment (codingllama) [inline lib/backend/report.go:412]:
```suggestion
	// Global passwordless challenges, keyed by challenge, as per
	// https://github.com/gravitational/teleport/blob/01775b73f138ff124ff0351209d629bb01836869/lib/services/local/users.go#L1510.
	"sessionData",
```

File: lib/backend/report_test.go (hunk 1 above)
Comment (codingllama) [inline lib/backend/report_test.go:97]: nit:

```suggestion
		{"/webauthn/sessionData/login/1234-5678", "/webauthn/sessionData"},
```

**Review by codingllama — APPROVED** [review]
Many thanks!

**Comment by codingllama** [comment]
> This PR needs to be rebased and merged instead of squash to retain the commits because:

You can merge and squash, just keep the branch around until you create the backports, as you can cherry-pick the commits directly from the branch (which, I believe, is what the backport bot does).

File: lib/backend/report.go [hunk 3]
```diff
@@ -408,8 +408,8 @@ func buildKeyLabel(key string, sensitivePrefixes []string) string {
...
 	"access_requests",
```
Comment (zmb3) [inline lib/backend/report.go:413]: A mix of camelCase and snake_case in the backend is not ideal. Bummer 😿

**Review by zmb3 — APPROVED** [review]

**Pull request merged**